src/main/java/com/vmware/vipclient/i18n/messages/api/opt/server/ComponentBasedOpt.java

```diff
@@ -81,10 +78,6 @@ private JSONObject getMsgsJson(Map<String, Object> response) {
 			if (null == responseStr || responseStr.equals(""))
 				return null;
 			else {
-				if (ConstantsKeys.LATEST.equals(this.dto.getLocale())) {
-					responseStr = responseStr.replace(ConstantsKeys.PSEUDOCHAR, "");
-				}
-	
```

Comment:
Could you explain what problem do these lines resolve before you remove them?

Reply:
Same comment as above.